packages/server/src/controllers/bookings.ts

```diff
@@ -49,20 +51,48 @@ export const createBooking = async (ctx: Context) => {
 export const getBooking = async (ctx: Context) => {
   const { bookingId } = ctx.params;
   try {
-    const booking = await getBookingById(bookingId);
+    const booking = await retrieveBookingById(bookingId);
     ctx.body = booking;
   } catch (error) {
     console.log(`Error while retrieving a booking: ${bookingId}`);
     ctx.status = 500;
   }
 };
 
+export const getBookingsByInterval = async (ctx: Context) => {
+  try {
+    const { start, end } = ctx.query;
+
+    if (!start || !end) {
+      console.log(`Could not find query parameters`);
+      ctx.status = 500;
+    }
+
+    const formattedStart = format(new Date(start), "y/MM/dd");
+
+    const formattedEnd = format(new Date(end), "y/MM/dd");
+
+    const bookings = await retrieveBookingsByInterval(
+      formattedStart,
+      formattedEnd,
+    );
+
+    ctx.body = bookings;
+    ctx.status = 200;
+  } catch (error) {
+    console.log(error);
+    console.log(`Error while retrieving bookings by interval`);
+    ctx.status = 500;
+  }
+};
+
 export const getBookings = async (ctx: Context) => {
   try {
-    const bookings = await getAllBookings();
+    const bookings = await retrieveAllBookings();
```

Comment:
Why this renaming? What value does it bring ?

Reply:
So it's in order to differentiate handler functions from database functions :)) 
For example for the route returning all the bookings inside an interval, the controller function is named getBookingsByInterval, this function performing some logic and calling another function retrieveBookingsByInterval which will fetch the bookings in the DB. I can't really name both functions with the same name so I use get* for controller functions and retrieve* for DB functions :))